packages/regal/lib/src/ledger_theme/widgets/ledger_bottom_sheet.dart

```diff
@@ -33,7 +34,7 @@ Future<T?> showLedgerBottomSheet<T>({
       builder: (dialogContext) => Theme(
         data: dialogContext.theme,
         child: Padding(
-          padding: Spacing.lds200.y + Spacing.lds250.x,
+          padding: padding ?? Spacing.lds200.y + Spacing.lds250.x,
```

Comment:
@kannan-atoa keep the Spacing addition in parenthesis. 
`(Spacing... + Spacing...)`